toplevel/coqargs.ml

```diff
@@ -10,8 +10,9 @@
 
 let warning s = Flags.(with_option warn Feedback.msg_warning (Pp.strbrk s))
 
-let fatal_error ?extra exn =
-  Topfmt.print_err_exn ?extra exn;
+let fatal_error ?extra e =
+  Topfmt.print_err_exn ?extra e;
+  let exn, info = e in
```

Comment:
Why this change?

Reply:
For bad reasons: I did not find how to add an info to an exception without reraising this exception. Otherwise said, how to build an `exn` from an `iexn`?